src/NUnitFramework/framework/Internal/RuntimeFramework.cs

```diff
@@ -380,33 +387,41 @@ private static bool IsNetCore()
             return false;
         }
 
+#if NETSTANDARD2_0
         [MethodImpl(MethodImplOptions.NoInlining)]
         private static bool IsNetCore_Internal()
         {
-#if NETSTANDARD2_0
             // Mono versions will throw a TypeLoadException when attempting to run any method that uses RuntimeInformation
             // so we wrap it in a try/catch block in IsNetCore to catch it in case it ever gets this far
             if (System.Runtime.InteropServices.RuntimeInformation.FrameworkDescription.StartsWith(".NET Core", StringComparison.OrdinalIgnoreCase))
             {
                 return true;
             }
-#endif
+
             return false;
         }
+#endif
 
         private static bool IsRuntimeTypeName(string name)
         {
-            return Enum.GetNames( typeof(RuntimeType)).Any( item => item.ToLower() == name.ToLower() );
+            return Enum.GetNames(typeof(RuntimeType)).Contains(name, StringComparer.OrdinalIgnoreCase);
+        }
+
+        private static string GetShortName(RuntimeType runtime, Version version)
+        {
+            return runtime == RuntimeType.NetFramework || (runtime == RuntimeType.NetCore && version.Major >= 5)
+                ? "Net"
+                : runtime.ToString();
```

Comment:
This is to maintain backwards compatibility, right?

I actually tried to work out if it would matter, and from what I can see, this property is only ever set, and never used. But this fits with the minimal-change idea. 🙂

Reply:
I think this change is demanded by the tests. `net5.0` is `.NETCoreApp` while `net4.8` is `.NETFramework`, in terms of how the .csproj SDK works, and our platform includes/excludes should be consistent with that.

Reply:
You could be right though that the tests could be changed without confusing our users.